### DISCUSSION TITLE
Since 45 Minutes I see no progress in Poetry

### DISCUSSION THREAD

**Discussion by dschiller** [python-poetry/poetry#7554] [General]
Is there a trick to force poetry do something ?

Since 45  minutes I see the following picture in the terminal:

<img width="1203" alt="Screenshot 2023-02-24 at 16 37 37" src="https://user-images.githubusercontent.com/3144986/221220603-6660ca7d-1b8f-4fd1-854e-2b6b32f8fcde.png">

I am using Python 3.8.2 and poetry 1.1.15 on macOS 10.15.7.

**Comment by TheErk** [comment]
Did you try a more recent version of poetry like 1.3.x ?

Note however that you seem to be using python 2.7 somehow (may be in your venv or in your requirement) because poetry warns you about it.

Python 2.7 is not supported anymore (not only by poetry but but python project itself), see: https://devguide.python.org/versions/#versions.
